Annotation/polar_golden_set.json

```diff
@@ -294,19 +294,19 @@
             {
                 "predicate": [
                     {
-                        "str": "curious",
+                        "str": "'re curious",
                         "lemma": "",
                         "POS": ""
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether we can provide you with a more affordable alternative",
                 "embedded clauses": []
             }
         ]
     },
     "74": {
-        "sentence": "[39] As evidence of this, In Thomas de Quinceys Confessions of an English Opium-Eater (1821, p. 188), it is not Ottoman, nor Chinese, addicts about whom he writes, But English opium users: \"I question whether any Turk, of all that ever entered the paradise of opium-eaters, can have had half the pleasure I had.\"",
+        "sentence": "As evidence of this, In Thomas de Quinceys Confessions of an English Opium-Eater (1821, p. 188), it is not Ottoman, nor Chinese addicts about whom he writes, But English opium users: \"I question whether any Turk, of all that ever entered the paradise of opium-eaters, can have had half the pleasure I had.\"",

```

Comment:
I'm not sure why you removed the [39]. Are we intending to remove all citation things like that in the sentences?

Reply:
Yeh, I wanted to ask if you think we should be cleaning the sentences of stuff like this?